src/randomizer.py

```diff
@@ -2007,46 +2044,45 @@ def visit_Decl(self, decl: ca.Decl) -> None:
 
 
 class Randomizer:
-    def __init__(self, rng_seed: int) -> None:
+    def __init__(self, rng_seed: int, dir: str) -> None:
```

Comment:
randomizer.py should not read from disk: it's better to centralize that responsibility to a single piece of code that reads a nonmatching directory, and furthermore it doesn't work with p@h.

Reply:
done